src/main/resources/project/lib/EC/Plugin/Core.pm

```diff
@@ -1,5 +1,5 @@
 #
-#  Copyright 2015 Electric Cloud, Inc.
+#  Copyright 2017 Electric Cloud, Inc.
```

Comment:
Actually you may leave the year be, according to https://www.copyrightlaws.com/copyright-notice-year/